src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIfStatement.java

```diff
@@ -32,60 +32,60 @@ private void method() {
             return;
         }
 
-        if ((x == y)           // violation
-                || (x != z)) { // violation
+        if ((x == y)           // violation 'Unnecessary parentheses around expression'
+                || (x != z)) { // violation 'Unnecessary parentheses around expression'
             return;
         }
 
-        if ((                       // violation
-                (x == y)            // violation
-                        || (x != z) // violation
+        if ((                       // violation 'Unnecessary parentheses around expression'
+                (x == y)            // violation 'Unnecessary parentheses around expression'
+                        || (x != z) // violation 'Unnecessary parentheses around expression'
         )) {
             return;
         }
 
-        if ((Integer.valueOf(x) instanceof Integer) // violation
+        if ((Integer.valueOf(x) instanceof Integer) // violation '.* parentheses around expression'
                 || Integer.valueOf(y) instanceof Integer) { // ok
             return;
         }
         if (x == ((y<z) ? y : z) &&
-            ((x>y && y>z)                  // violation
-                    || (!(x<z) && y>z))) { // violation
+            ((x>y && y>z)                  // violation 'Unnecessary parentheses around expression'
+                    || (!(x<z) && y>z))) { // violation 'Unnecessary parentheses around expression'
             return;
         }
-        if ((x >= 0 && y <= 9)            // violation
-                 || (z >= 5 && y <= 5)    // violation
-                 || (z >= 3 && x <= 7)) { // violation
+        if ((x >= 0 && y <= 9)            // violation 'Unnecessary parentheses around expression'
+                 || (z >= 5 && y <= 5)    // violation 'Unnecessary parentheses around expression'
+                 || (z >= 3 && x <= 7)) { // violation 'Unnecessary parentheses around expression'
             return;
         }
         if(x>= 0 && (x<=8 || y<=11) && y>=8) { // ok
             return;
         }
-        if((y>=11 && x<=5)            // violation
-                || (x<=12 && y>=8)) { // violation
+        if((y>=11 && x<=5)            // violation 'Unnecessary parentheses around expression'
+                || (x<=12 && y>=8)) { // violation 'Unnecessary parentheses around expression'
             return;
         }
     }
     private void check() {
         String sectionName = "Some String";
         if ("Some content".equals(sectionName) || "Some overview".equals(sectionName) // ok
-                || (!"AbbreviationAsWordInName".equals(sectionName) // violation
-                                                                    // parenthesis
+                || (!"A".equals(sectionName) // violation 'Unnecessary paren.* around expression'
+                                             // parentheses
                 && !"AbstractClassName".equals(sectionName) // ok
```

Comment:
Remove content from this line, it should be empty line.

Reply:
Is there a reason to keep it empty? Why can't we remove it?

Reply:
If you remove it, it will force you to shift all line number in test method expected array.
I just try to avoid extra work for you and extra changes for me to review

Reply:
Thanks! But if it is okay with you, can I remove those two empty lines?

Reply:
Ok to remove

Reply:
Done